src/content/docs/style-guide/structure/styleguide-markup-indentation.mdx

```diff
@@ -0,0 +1,324 @@
+3. Incident policy review: Using the AQM dashboard, you and the stakeholders will identify the noisiest incident policies. Once identified, those policies should be evaluated as detailed in step 4 below.
+4. Alert policy recommendations: In this step, you and the stakeholders will review the noisiest policies using the following criteria:
+  * Do the alerts have any business impact?
+  * Are the policies properly configured?
+    * Are they telling us something about the resource that needs to be fixed?
+    * Are the policies necessary? Do they have business impact?
+    * Are the thresholds set properly?
+5. Technical recommendations: Here, you and the stakeholders will review any technical recommendations, including:
+    * Are there application / system problems for engineering to review?
+    * Are there poorly constructed policies that need to be fixed?
+    * Are there instrumentation gaps?
+```
+This Markdown with two-space indentation renders like this:
+
+<img
+  title="Screenshot showing two-space nesting problem"
+  alt="Screenshot showing two-space nesting problem"
+  src={nestingBulletListOne}
+/>
+
+Here is the same Markdown content but with four spaces preceding the bullets:
+
+```
+The process consists of four activities:
+1. Review AQM dashboard and KPI trends: Here you and the stakeholders will look at the AQM KPIs and identify their week over week trends. The team should identify areas where KPIs are not improving and develop strategies to drive improvement.
+2. Identify achievements, challenges, and opportunities: Here you and the stakeholders will map the current state of alert quality to business impact, identifying areas where improvement has resulted in better business outcomes and areas where problems are impacting business outcomes.
+3. Incident policy review: Using the AQM dashboard, you and the stakeholders will identify the noisiest incident policies. Once identified, those policies should be evaluated as detailed in step 4 below.
+4. Alert policy recommendations: In this step, you and the stakeholders will review the noisiest policies using the following criteria:
+    * Do the alerts have any business impact?
+    * Are the policies properly configured?
+        * Are they telling us something about the resource that needs to be fixed?
+        * Are the policies necessary? Do they have business impact?
+        * Are the thresholds set properly?
+5. Technical recommendations: Here, you and the stakeholders will review any technical recommendations, including:
+    * Are there application / system problems for engineering to review?
+    * Are there poorly constructed policies that need to be fixed?
+    * Are there instrumentation gaps?
+```
+
+The Markdown with four space indentation renders like this:
+
+<img
```

Comment:
Great section! Switching to screenshots makes it a lot easier to skim and get the "gist" of what's going on. This is a great example of screenshots being just as important as your exposition/descriptive text.

Reply:
I also like the alternating indention for your images too. Makes it very clear, distinctive, and also demonstrates what you're writing about in practice + on the page. A+